homework1.py

```diff
@@ -0,0 +1,134 @@
+from typing import List
+from pydantic import BaseModel
+from abc import ABC, abstractmethod
+
+# Pydantic модель для книги
+class BookModel(BaseModel):
+    title: str
+    author: str
+    year: int
+
+    def __str__(self):
+        return f"{self.title} ({self.author}, {self.year})"
+
```

Comment:
There should be 2 indents between classes and functions due to PEP-8 rules. Please fix it everywhere in your code